packages/block-library/src/navigation-menu-item/edit.js

```diff
@@ -92,6 +105,25 @@ function NavigationMenuItemEdit( {
 
 	return (
 		<Fragment>
+			<BlockControls>
+				<Toolbar>
+					<ToolbarButton
+						name="link"
+						icon="admin-links"
+						title={ __( 'Link' ) }
+						onClick={ () => setIsLinkOpen( ! isLinkOpen ) }
+					/>
+					{ isLinkOpen &&
+						<LinkControl
+							className="wp-block-navigation-menu-item__inline-link-input"
+							onKeyDown={ handleLinkControlOnKeyDown }
+							onKeyPress={ ( event ) => { event.stopPropagation() } }
+							onClose={ onFocusOutside }
+							fetchSearchSuggestions={ fetchSearchSuggestions }
```

Comment:
`fetchSearchSuggestions` shouldn't be required, looking at the code for `LinkControl` it should be optional. I think it's a bug that needs to be fixed in `LinkControl` if it is required.

Reply:
Looks like it's not required.

Reply:
I've just added this option last night! :-D